pkg/reconciler/namespace/controller.go

```diff
@@ -132,6 +145,11 @@ type Controller struct {
 	kubeClient      kubernetes.ClusterInterface
 	ddsif           informer.DynamicDiscoverySharedInformerFactory
 	gvkTrans        *gvk.GVKTranslator
+
+	// Mapping of namespace key to the last scheduling decision for
```

Comment:
"last scheduling decision" = a pcluster name?

Reply:
For now, yes.